src/main/java/org/codeforamerica/messaging/providers/mailgun/MailgunCallbackController.java

```diff
@@ -55,18 +53,14 @@ public ResponseEntity<Object> updateStatus(@RequestBody JsonNode requestJSON) {
     private void enqueueStatusUpdate(JsonNode requestJSON, String providerMessageId,
             String rawEmailStatus) {
         MessageStatus newEmailStatus = mapMailgunStatustoMessageStatus(rawEmailStatus);
-        Map<String, String> providerError = hadError(newEmailStatus)? buildProviderError(requestJSON, newEmailStatus): null;
-        JobId id = jobRequestScheduler.enqueue(new EmailMessageStatusUpdateJobRequest(providerMessageId, rawEmailStatus, newEmailStatus, providerError));
+        Map<String, String> providerError = hadError(newEmailStatus) ? buildProviderError(requestJSON, newEmailStatus) : null;
+        jobRequestScheduler.enqueue(
+                new EmailMessageStatusUpdateJobRequest(providerMessageId, rawEmailStatus, newEmailStatus, providerError));
     }
 
     private void unsubscribeEmail(JsonNode requestJSON) {
         String unsubscribedEmail = requestJSON.at("/event-data/recipient").textValue();
-        log.info("Unsubscribing");
-        emailSubscriptionRepository.save(EmailSubscription.builder()
-                .email(unsubscribedEmail)
-                .sourceInternal(true)
-                .unsubscribed(true)
-                .build());
+        emailService.unsubscribe(unsubscribedEmail);
```

Comment:
It's my opinion that this method could be one lined instead to reduce the number of chained methods and still be readable, eg replace line 36 with:
```
emailService.unsubscribe(requestJSON.at("/event-data/recipient").textValue());
```

but not important to change it

Reply:
I agree. The method made sense when it used to be longer.